src/Discord/Parts/Guild/Guild.php

```diff
@@ -635,9 +635,7 @@ public function getVoiceRegions(): ExtendedPromiseInterface
      */
     public function createRole(array $data = [], ?string $reason = null): ExtendedPromiseInterface
     {
-        $botperms = $this->members->get('id', $this->discord->id)->getPermissions();
-
-        if ($botperms && ! $botperms->manage_roles) {
+        if (! $this->getBotPermissions()->manage_roles) {
```

Comment:
The `$botperms` was assigned to another variable because the getBotPermissions() can return null in case of missing cache, which prevented `null->manage_roles` error.